aviary/aviary.py

```diff
@@ -153,6 +153,13 @@ def main():
         default=8,
     )
 
+    base_group.add_argument(
+        '-p', '--pplacer-threads', '--pplacer_threads',
+        help='The number of threads given to pplacer, values above 48 will be scaled down',
+        dest='pplacer_threads',
+        default=48,
```

Comment:
This default value is really high, I think previous it defaulted to the number of standard threads allocated unless a specific amount of pplacer threads were set. 48 might be too high, it will cause some peoples jobs to crash due to OOM and they might get confused

Reply:
It is still restricted to a max of args.max_threads in processor (https://github.com/rhysnewell/aviary/blob/0023711945aa89e8e249cbfd055141c73fad419d/aviary/modules/processor.py#L106). But maybe defaulting to 16 is better for when they request 48 max_threads?